src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/DiagnosticComputer.cs

```diff
@@ -307,11 +307,11 @@ static async Task WaitForHighPriorityTasksAsync(CancellationToken cancellationTo
                         if (task.IsCompleted)
                         {
                             // Make sure to yield so continuations of 'task' can make progress.
-                            await Task.Yield().ConfigureAwait(false);
+                            await AwaitExtensions.ConfigureAwait(Task.Yield(), false);
```

Comment:
curious why it needs to be called in extension form.

Reply:
➡️ Roslyn has the same extension defined in Roslyn.Utilities, and unlike how we handled `NoThrowAwaitable` it wasn't renamed to `ConfigureAwaitInternal`.

Reply:
can we remove that though, and just use the new extension instead?  (this can all come in a later PR of coursE).

Reply:
➡️ Not trivially. It could be renamed, but this was the only potential conflict location. There are several references in workspaces/features where the conflict can't occur.